server/hotrod/src/main/resources/META-INF/services/org.infinispan.factories.components.ModuleMetadataFileFinder

```diff
@@ -0,0 +1 @@
+org.infinispan.server.hotrod.ServerMetadataFileFinder
```

Comment:
+1 @anistor. We don't even need a special marker value for this, the user can either use the overload without a segment list or he can use the set `{0..numSegments-1}`.